tools/wasm-sourcemap.py

```diff
@@ -224,14 +224,26 @@ def extract_comp_dir_map(text):
   comp_dir_pattern = re.compile(r"DW_AT_comp_dir\s+\(\"([^\"]+)\"\)")
 
   map_stmt_list_to_comp_dir = {}
-  chunks = compile_unit_pattern.split(text) # DW_TAG_compile_unit
-  for chunk in chunks[1:]:
-    stmt_list_match = stmt_list_pattern.search(chunk) # DW_AT_stmt_list
+  iterator = compile_unit_pattern.finditer(text)
+  try:
+    current_match = next(iterator)
+  except StopIteration:
+    return map_stmt_list_to_comp_dir
```

Comment:
Can you skip the try/catch and just do `current_match = next(iterator, None)`?

Reply:
Done: https://github.com/emscripten-core/emscripten/pull/25935/commits/08a8402de41ac19f3cb99a2d6735408807b7a487 and https://github.com/emscripten-core/emscripten/pull/25935/commits/534b868ce3bc94ed683bf4fe6308f56168a6c169